upstream-community-operators/synopsys/synopsys-operator.2019.4.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,479 @@
+    mediatype: image/png
+  keywords: [security monitoring]
+  version: 2019.4.0
+  maturity: stable
+  maintainers:
+  - name: Synopsys, Inc
+    email: software-integrity-support@synopsys.com
+  provider:
+     name: Synopsys, Inc
+  labels:
+     alm-owner-etcd: synopsys-operator
+     alm-status-descriptors: synopsys-operator.2019.4.0
+  selector:
+     matchLabels:
+       alm-owner-etcd: synopsys-operator
+       alm-status-descriptors: synopsys-operator.2019.4.0
+  links:
+   - name: Synopsys Integrity
+     url: https://www.synopsys.com/software-integrity.html
+   - name: Synopsys Operator Documentation
+     url: https://synopsys.atlassian.net/wiki/spaces/BDLM/pages/34373652/Synopsys+Operator
+   - name: Black Duck Datasheet
+     url: https://www.synopsys.com/content/dam/synopsys/sig-assets/datasheets/blackduck-sca-ds-ul.pdf
+   - name: OpsSight Data Sheet
+     url: https://www.synopsys.com/content/dam/synopsys/sig-assets/datasheets/openshift-ds-ul.pdf
+   - name: Black Duck Alert Documentation
+     url: https://synopsys.atlassian.net/wiki/spaces/INTDOCS/pages/622882/Black+Duck+Alert
+   - name: Synopsys Operator Source Code
+     url: https://github.com/blackducksoftware/synopsys-operator
+  installModes:
+   - type: OwnNamespace
+     supported: true
+   - type: SingleNamespace
+     supported: false
+   - type: MultiNamespace
+     supported: false
+   - type: AllNamespaces
+     supported: true
+  install:
+    strategy: Deployment
+    spec:
+      deployments:
+        - name: synopsys-operator
+          spec:
+            replicas: 1
+            selector:
+              app: synopsys-operator
+              component: operator
+            template:
+              metadata:
+                labels:
+                  app: synopsys-operator
+                  component: operator
+                name: synopsys-operator
+                namespace: placeholder
+              spec:
+                containers:
+                - args:
+                  - /etc/synopsys-operator/config.json
+                  command:
+                  - ./operator
+                  env:
+                  - name: SEAL_KEY
```

Comment:
Is there any documentation for this `SEAL_KEY`? Could this be added to the description if this is required for the operator to run?

Reply:
Yes, documentation will be soon published in our external docs site, which is referenced in the links section.